packages/next/src/server/app-render/app-render.tsx

```diff
@@ -1071,11 +1071,18 @@ async function renderToHTMLOrFlightImpl(
     formState,
   })
 
+  const requestStoreWaitUntilTasks = requestStore.waitUntil.map((p) =>
```

Comment:
To clarify: this means that the callbacks cannot add additional `waitUntil` calls?